src/components/atoms/Input.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+interface Props {
+    type: string,
```

Comment:
mets un ENUM plutot  que stirng comme ça il nous proposera les choix avec TS 

genre tu fais ça
```suggestion
    type: 'text' | 'password',
```